src/andromede/simulation/runner.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+import os
+import pathlib
+import subprocess
+import sys
+from typing import List
+
+
+class CommandRunner:
+    current_dir: pathlib.Path
```

Comment:
If not a `dataclass`, those class-level fields are not used --> they must be removed

Reply:
Ok, I will change it, but as I said [here](https://github.com/AntaresSimulatorTeam/andromede-modeling-prototype/pull/12#discussion_r1498887384), if they don't have default values, they are just annotations and not actuel class-fields

Reply:
Sorry I missed your previous comment, I've been wrong the whole time :) 
My python knowledge is not up to date ! Thanks for sharing the reference.